src/model/HumidifierSteamGas.cpp

```diff
@@ -343,6 +344,22 @@ namespace model {
       }
     }
 
+    ComponentType HumidifierSteamGas_Impl::componentType() const {
+      return ComponentType::Heating;  // TODO: is that right?
+    }
+
+    std::vector<FuelType> HumidifierSteamGas_Impl::coolingFuelTypes() const {
+      return {};
+    }
+
+    std::vector<FuelType> HumidifierSteamGas_Impl::heatingFuelTypes() const {
+      return {FuelType::Gas};  // TODO: is that right?
```

Comment:
I'd say it's pretty clearly a natural gas device so you can class it as heating with fuel type gas / fuel

Reply:
My doubt is more about whether a Humidifier should be considered a heating element.